### ISSUE TITLE
Incompatible W3C patterns between Ruby bindings and server

### ISSUE THREAD

**Issue by xuanzhaopeng** [SeleniumHQ/selenium#4641]
## Meta -
OS:  
MACOS Darwin
Selenium Version:  
Ruby bindings: 3.5.1 
Standalone server: 3.5.1
Browser:  
IOS Simulator safari

## Error
```
java.lang.IllegalArgumentException: W3C payload contained keys that do not comply with the spec: [platformVersion]
	at org.openqa.selenium.remote.server.NewSessionPayload.validateSpecCompliance(NewSessionPayload.java:179)
```

## Step to reproduce:
1) Start selenium standalone server as hub
2) Start appium and register to hub
3) Start test with following config by appium ruby bindings
```
    automationName: 'xcuitest'
    platformName: 'ios'
    platform: :mac
    platformVersion: '10.3'
    browserName: 'safari'
    deviceName: 'iPhone 6s'
    newCommandTimeout: 1200
    marionette: false
    autoAcceptAlerts: false
    autoDismissAlerts: true
    showIOSlogs: true
    noReset: true
```

In step 3, in fact appium will call selenium ruby binding W3C::Capabilities.from_oss to generate w3c capabilities and to do a handshake(https://github.com/SeleniumHQ/selenium/blob/master/rb/lib/selenium/webdriver/remote/bridge.rb#L183)

The W3C defined platform_version is a KNOWN w3c pattern: https://seleniumhq.github.io/selenium/docs/api/rb/Selenium/WebDriver/Remote/W3C/Capabilities.html

However in selenium standalone server side, you don't define 'platformVersion' is a valid w3c patterns
https://github.com/SeleniumHQ/selenium/blob/master/java/server/src/org/openqa/selenium/remote/server/NewSessionPayload.java#L63

In W3C definition: https://www.w3.org/TR/webdriver/, "PlatformVersion" is not defined. So could u please remove it from Selenium Ruby bindings.

Thanks

**Comment by xuanzhaopeng** [comment]
I make a PR for fixing this problem and test it locally works fine with latest appium(ruby binding)

**Comment by p0deje** [comment]
Fixed in 3.5.2.
